src/containers/pages/MDAPoint/ClientListView/helpers/serviceHooks.ts

```diff
@@ -12,12 +12,19 @@ import store from '../../../../../store';
 import { fetchFiles, File } from '../../../../../store/ducks/opensrp/clientfiles';
 import { getAccessToken } from '../../../../../store/selectors';
 
+/** useState function type for setting loader on and off */
+type SetIsLoading = ((status: boolean) => void) | null;
+
 /** loads and persists to store files data from upload/history endpoint
  */
 export const loadFiles = async (
+  setLoading: SetIsLoading = null,
```

Comment:
@ciremusyoka lets make this totally optional, in the sense that we can call loadFiles and not need to have to pass this.